charts/pulsar-operator/crds/pulsar.streamnative.io_pulsarproxies.yaml

```diff
@@ -11,1939 +11,1941 @@ spec:
-          name: Age
-          type: date
-      name: v1alpha1
-      schema:
-        openAPIV3Schema:
-          properties:
-            apiVersion:
-              type: string
-            kind:
-              type: string
-            metadata:
-              type: object
-            spec:
-              properties:
-                apiObjects:
-                  properties:
-                    externalService:
-                      properties:
-                        exposeSSLPort:
-                          type: boolean
-                        metadata:
-                          properties:
-                            annotations:
-                              additionalProperties:
-                                type: string
-                              type: object
-                            labels:
-                              additionalProperties:
-                                type: string
-                              type: object
-                            name:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.replicas
+      name: Replicas
+      type: integer
+    - jsonPath: .status.replicas
+      name: Ready Replicas
+      type: integer
+    - jsonPath: .spec.image
+      name: Desired Image
+      type: string
+    - jsonPath: .spec.dnsNames[0]
+      name: Endpoint
+      type: integer
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              apiObjects:
+                properties:
+                  externalService:
+                    properties:
+                      exposeSSLPort:
+                        type: boolean
+                      metadata:
+                        properties:
+                          annotations:
+                            additionalProperties:
                               type: string
-                          type: object
-                        ports:
-                          items:
-                            properties:
-                              name:
-                                type: string
-                              nodePort:
-                                format: int32
-                                type: integer
-                              port:
-                                format: int32
-                                type: integer
-                              targetPort:
-                                format: int32
-                                type: integer
                             type: object
-                          type: array
-                        type:
-                          type: string
-                        updatePolicy:
-                          items:
+                          labels:
```

Comment:
pulsarproxy crd seems to have a lot of changes that are not expected, am I missing something? Please take a look at this pr when you have time, thanks @fantapsody @MarvinCai

Reply:
Most changes are indent related, it's ok